tokio/Cargo.toml

```diff
@@ -108,7 +105,7 @@ num_cpus = { version = "1.8.0", optional = true }
 parking_lot = { version = "0.12.0", optional = true }
 
 [target.'cfg(not(any(target_arch = "wasm32", target_arch = "wasm64")))'.dependencies]
```

Comment:
We can also replace `any(target_arch = "wasm32", target_arch = "wasm64")` with `target_family = "wasm"`.

Reply:
Thanks!
I've fixed this.